src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -353,6 +353,7 @@ public void SendFunctionEnvironmentReloadRequest()
 
         internal void SendFunctionLoadRequest(FunctionMetadata metadata)
         {
```

Comment:
Oh, also, we should log when we send this load request and when we receive the response (with timing). It'd help us see how long this takes...

Reply:
Good idea. Added logs.